src/components/taskDetails/ProgressUpdateCard/ProgressUpdateCard.tsx

```diff
@@ -35,27 +69,38 @@ export default function ProgressUpdateCard({
         setIsTooltipVisible(false);
     }
 
-    function onMoreButtonClick(e: MouseEvent<HTMLElement>) {
-        if (isTitleWrapped) {
-            setTitleToShow(data.completed);
-            setIsTitleWrapped((prevState) => !prevState);
-            return;
-        }
+    function onMoreOrLessButtonClick(
+        e: MouseEvent<HTMLElement>,
+        clickedOnData: ProgressUpdateDataToShow
+    ) {
+        e.stopPropagation();
+        setDataToShowState((prevState) => {
+            const newPrevState = prevState.map((datum) => {
+                if (datum.id === clickedOnData.id) {
+                    datum.isReadMoreEnabled = !datum.isReadMoreEnabled;
+                }
+                return datum;
+            });
+
+            return newPrevState;
+        });
+    }
```

Comment:
Now we have collapsable elements I don't think **More** and **Less** buttons are still required.

Reply:
I will take Ankush's feedback then I will remove this, btw the point legit to me